docs/tutorials/benchmarking.ipynb

```diff
@@ -345,6 +345,7 @@
    "mimetype": "text/x-python",
    "name": "python",
    "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
```

Comment:
This line causes nbsphinx to crash on my laptop. Not sure if that's because there's a bug in one of my Spack packages or what. Also not sure if it makes any difference.

Reply:
Yeah I have no idea what these all mean